code/modules/mining/ore.dm

```diff
@@ -16,6 +16,31 @@
 			user << "<span class='info'>Not enough fuel to smelt [src].</span>"
 	..()
 
+/obj/item/weapon/ore/Crossed(AM as mob|obj)
+	var/obj/item/weapon/storage/bag/ore/OB
+	var/turf/simulated/floor/F = get_turf(src)
+	if(loc != F)
+		return ..()
+	if(ishuman(AM))
+		var/mob/living/carbon/human/H = AM
```

Comment:
I don't really know code, so please excuse me if I'm wrong, but just glancing at this, is the requirement for this feature to be a human or will this apply to all races? I'm just double checking to make sure this ends up correct.